packages/scene-graph-mediator/client/src/modules/fillText.ts

```diff
@@ -0,0 +1,51 @@
+export { };
+
+declare global {
+  interface String {
+    fillRight(word: string, count: number): string;
+    fillLeft(word: string, count: number): string;
+  }
+}
+String.prototype.fillLeft = function (word: string, count: number): string {
```

Comment:
Extending native api is not a good way.
Consider implement this function as independent module, or using npm package like
https://www.npmjs.com/package/pad

Reply:
Method name `fillLeft` or `fillRight` has no semantic of slicing.
Some user does not expect input string is cut.